routes/lobby_api.js

```diff
@@ -0,0 +1,6 @@
+exports = module.exports = function(req, res) {
+    const request = require('request');
+    request(process.env.LOBBY_API_URL + '/'+req.query.resource, function (error, response, body) {
+        res.send(JSON.parse(body));
+    });
+};
```

Comment:
Do I understand right, that we're asking our website to ask the api, not the api directly?
Is there any reason to create such unnecessary load?

Reply:
CORS Policy